### DISCUSSION TITLE
Launching Android Studio from WSL2

### DISCUSSION THREAD

**Discussion by leo-petrucci** [ionic-team/capacitor#4848] [Q&A]
I've been banging my head against this wall all morning and I'm completely lost.

Is there any way to launch a capacitor project in WSL to the Windows installation of Android studio?

[There used to be an option that allowed this](https://github.com/ionic-team/capacitor/issues/3186), but all the threads I've found have been closed with the message:

> linuxAndroidStudioPath config option no longer exists on capacitor 3, closing

I tried setting `CAPACITOR_ANDROID_STUDIO_PATH` to `/mnt/c/Program Files/Android/Android Studio/bin/studio64.exe`, which if run in WSL opens my project, but doesn't seem to do anything after running `npx cap open android`.

```
printenv CAPACITOR_ANDROID_STUDIO_PATH
/mnt/c/Program Files/Android/Android Studio/bin/studio64.exe
```

I've tried starting Android Studio in a virtual window manager from WSL, but it doesn't recognise any devices (not even real ones.

I've tried starting Windows Android Studio, but it won't load files from WSL as they're network mounted.

Has anyone been able to find a solution to this? Developing Node on Windows is an absolute pain and I'd rather keep it all on WSL2.

**Comment by 8483** [comment]
Bump. I also want to know how to use capacitor in WSL.

**Reply by jcesarmobile** [reply]
the creator of the discussion created this package 
https://github.com/leo-petrucci/run-cap-on-android

**Reply by buzzy** [reply]
Sorry, why is all this needed? Why can't you just mount WSL2 to a drive letter in windows and open the project? Works fine for me?
